Daily-Weather/Services/Remote/APIUrl.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  APIUrl.swift
+//  Daily-Weather
+//
+//  Created by Nguyen Tien Cong on 8/28/20.
+//  Copyright © 2020 Nguyen Tien Cong. All rights reserved.
+//
+
+import Foundation
+import CoreLocation
+
+struct APIUrl {
+    static var api = APIUrl()
+    private let baseUrl : String = "https://api.openweathermap.org/data/2.5"
+}
+
+extension APIUrl {
+    public func Url( lat : CLLocationDegrees, lon : CLLocationDegrees ) -> String {
+        var url : String = baseUrl
+        url = url + "/onecall?lat=\(String(lat))&lon=\(String(lon))&%20exclude=daily&appid=051eccdec971db6541e789fd524cfb66"
```

Comment:
```suggestion
        let url = baseUrl + "/onecall?lat=\(String(lat))&lon=\(String(lon))&%20exclude=daily&appid=051eccdec971db6541e789fd524cfb66"
```
để key sang 1 file riêng và ignore file đó đi em. Không đẩy key lên github